src/base/rss/rss_autodownloader.cpp

```diff
@@ -266,6 +270,38 @@ void AutoDownloader::importRulesFromLegacyFormat(const QByteArray &data)
         insertRule(AutoDownloadRule::fromLegacyDict(val.toHash()));
 }
 
+QStringList AutoDownloader::getSmartEpisodeFilters() const
+{
+    QVariant filtersSetting = SettingsStorage::instance()->loadValue(SettingsKey_SmartEpisodeFilter);
+
+    if (filtersSetting.isNull()) {
+        QStringList filters;
+        filters.append("(?:s(\\d+)e(\\d+))"); // Format 1: s01e01
```

Comment:
It's not so obvious for user to add outer non-capturing group (as well as to have any captures here). It can type just, e.g., "sn\d+ep\d+" and wonder why it not works properly. Actually, the user should understand how it works to set valid regex.

Reply:
I've tweaked the regex creation to insert a non-capturing group around each item. I was meaning to do that anyway. I've also modified these rules to not include a redundant non-capturing group.